client/js/backstage.js

```diff
@@ -157,29 +157,29 @@ var BackstageView = Backbone.View.extend({
         this.$openButton    = this.$el.find('.silky-bs-open-button')
         this.$saveButton    = this.$el.find('.silky-bs-save-button')
         this.$saveAsButton  = this.$el.find('.silky-bs-save-as-button')
-		
+
 		this.main = new BackstagePlaces({ el: ".silky-bs-main", model: this.model })
 	},
 	activate : function() {
-	
+
 		this.$el.fadeIn(100)
-		
```

Comment:
There was a bunch of tabs that i took out, i thought i was committing those. I couldn't find a setting to turn it off. I can start using a different editor.